EXOSIMS/Observatory/TwoStarShades_mission.py

```diff
@@ -0,0 +1,567 @@
+from EXOSIMS.Observatory.SotoStarshade import SotoStarshade
+import numpy as np
+import astropy.units as u
+import astropy.constants as const
+import scipy.optimize as optimize
+
+
+class TwoStarShades_mission(SotoStarshade):
```

Comment:
You are missing the docstring for this class.  Please see examples in the codebase and the documentation guide.  Your class must have  a docstring with all inputs (in the order they're given in the init) as well as all class attributes (in alphabetical order). These must include data types and descriptions.